locale.c

```diff
@@ -1235,7 +1235,7 @@ S_parse_LC_ALL_string(pTHX_ const char * string,
 
 #  ifdef PERL_LC_ALL_USES_NAME_VALUE_PAIRS
 
-    const char separator[] = ";";
```

Comment:
separator itself is no longer const, but you've maintained the const in other similar strings below (eg `longer`)